adafruit_esp32spi/adafruit_esp32spi_wsgiserver.py

```diff
@@ -216,3 +230,13 @@ def _get_environ(self, client):
             env[key] = value
 
         return env
+
+    def check_remote_ip(self):
+        sock_num = self._client_sock.socknum
+        remote_ip = _the_interface.get_remote_data(sock_num)
+        return _the_interface.pretty_ip(remote_ip)
+
+    def print_remote_ip(self):
```

Comment:
Is it a good idea to have print functions like this in a library? The user can just print the return from check_remote_ip()

Reply:
Where is the check_remote_ip() method?

Reply:
right above print_remote_ip()

Reply:
https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/blob/29900615c81eb9a63407fe87f56f2ec87a37fddd/adafruit_esp32spi/adafruit_esp32spi_wsgiserver.py#L234

Reply:
Just making sure you were referring to my method as I was not sure if you were referring to something already existing.

Reply:
yes -- I think print statements are normally only used for debug.

Reply:
The check_remote_ip() method is designed to allow forced control if needed otherwise the print method is just for echoing the connections.  Two separate functionalities.

Reply:
I'll leave it to others to comment. I'm not used to seeing prints in libraries but I'll defer to the other reviewers.

Reply:
I agree, prints generally don't belong in libraries unless the are behind a debug flag. If you wish to paint information from the library you can expose it with a method or property and display it from your user code